core/crawler.py

```
@@ -1,70 +1,67 @@
+        list: All unique links discovered during crawl
+    """
+
+    links = [] 
+    pagesToVisit = [url]
+    numberVisited = 0
+    while numberVisited < maxPages and pagesToVisit:
+        numberVisited += 1
+        url = pagesToVisit.pop()
+        try:
+            parser = LinkParser()
+            data, links = parser.getLinks(url)
```

Ruff (0.15.9)

[warning] Unpacked variable `data` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)